cloudslang-entities/src/main/java/io/cloudslang/lang/entities/bindings/InOutParam.java

```diff
@@ -24,9 +24,9 @@
 	private static final long serialVersionUID = -7712676295781864973L;
 
 	private String name;
-	private String expression;
+	private Serializable expression;
```

Comment:
Maybe we should change the member name from expression to value?
Before, we treated everything as expressions, so it was OK
Now it is actually a value, which might be an expression.
WDYT?